build.sbt

```diff
@@ -105,4 +106,34 @@ lazy val annotationProcessorTestProject: CompositeProject =
         Compile / javacOptions ++= Seq("-processor", "akka.javasdk.tooling.processor.ComponentAnnotationProcessor"))
   }
 
+lazy val akkaJavaSdkParent =
+  Project(id = "akka-javasdk-parent", base = file("akka-javasdk-parent"))
+    .enablePlugins(BuildInfoPlugin, Publish)
+    .disablePlugins(CiReleasePlugin) // we use publishSigned, but use a pgp utility from CiReleasePlugin
+    .settings(
+      name := "akka-javasdk-parent",
+      crossPaths := false,
+      scalaVersion := Dependencies.ScalaVersion,
+      pomPostProcess := { (node: Node) =>
+        // completely replace with our pom.xml
+        val pom = scala.xml.XML.loadFile(baseDirectory.value / "pom.xml")
```

Comment:
not using the sbt generated pom, but replacing with the file

alternatively we could try to use some from sbt, and inject more stuff with `pomExtra`, but I think that is just more complicated